includes/Admin/Settings_Screens/Messenger.php

```diff
@@ -50,95 +55,74 @@ public function __construct() {
 	 */
 	public function enqueue_assets() {
 
-		$tab = SV_WC_Helper::get_requested_value( 'tab' );
-
-		// only enqueue assets on this specific screen
-		if ( Admin\Settings::PAGE_ID !== SV_WC_Helper::get_requested_value( 'page' ) || ( $tab && self::ID !== $tab ) ) {
-			return;
-		}
-
-		wp_enqueue_script( 'wc-facebook-admin-settings-messenger', facebook_for_woocommerce()->get_plugin_url() . '/assets/js/admin/facebook-for-woocommerce-settings-messenger.min.js', [ 'jquery', 'iris', 'wc-enhanced-select' ], \WC_Facebookcommerce::VERSION );
+		// TODO: empty for now, until we add more robust Messenger settings {CW 2020-06-17}
```

Comment:
Should we remove this JS file from the repo as well?

Reply:
I opted to keep it, as we'll soon be revamping it.